src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String newLine = System.lineSeparator();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromFormatter = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToFormatter = LocalDate.parse(dateTo, formatter).plusDays(1);
+        int[] salaries = new int[names.length];
+        for (int i = 0; i < names.length; i++) {
+            for (String dateFinding: data) {
+                String[] dateSplit = dateFinding.split(" ");
```

Comment:
Cannot say I understand naming you use.
datE - це дата (календарна), datA - якісь дані, інформація про щось
```suggestion
            for (String record: data) {
                String[] splitRecord = record.split(" ");
```